lib/carlo.js

```diff
@@ -517,4 +519,68 @@ function enterTestMode() {
   testMode = true;
 }
 
+async function downloadChromium() {
+  const downloadHost = process.env.CARLO_DOWNLOAD_HOST || process.env.npm_config_carlo_download_host;
+
+  const browserFetcher = puppeteer.createBrowserFetcher({ host: downloadHost });
+
+  const revision = process.env.CARLO_CHROMIUM_REVISION || process.env.npm_config_carlo_chromium_revision
```

Comment:
I don't think it is safe to use anything outside `require('puppeteer-core/package.json').puppeteer.chromium_revision`, keep it hardcoded?

Reply:
done.